### DISCUSSION TITLE
Is there a way to use markdowns/references inside code blocks? If yes, how?

### DISCUSSION THREAD

**Discussion by ItsTheSky** [squidfunk/mkdocs-material#5511] [Q&A]
Hey there 👋 
I would like to incorporate bold and link text inside a code block (where the target text is part of a comment).
Is that even possible? And if yes, how can I make it work? Thanks!

**Comment by xsa** [comment]
You can use the `<code>` HTML tag, and insert whatever you want in there, e.g.

```
<code>
**This is bold text**

[This is a link](https://foo.bar.baz)
</code>
```

**Reply by ItsTheSky** [reply]
Gotcha, Thanks!
